src/STM32_BlueNRG/SimpleBlueNRG_HCI/hci/controller/bluenrg_IFR.c

```diff
@@ -0,0 +1,365 @@
+  0x5B,0x00,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+#endif /* BLUENRG_MS */
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+
+  0x02,0x1C,0x43,0x02,
+  0x20,0xEC,0x02,0x1F,
+  0xAF,0x00,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+
+  STACK_MODE,
+  0xFF,0xFF,0xFF,
+  0xFFFFFFFF,0xFFFFFFFF,0xFFFFFFFF,
+  0xFFFFFFFF,
+  htobl(0x00190000),
+  htobl(0x0028F5C2),
+  htobs(SLAVE_SCA_PPM),
+  MASTER_SCA,
+  0xFF,
+  htobs(FROM_US_TO_SYS_TIME(HS_STARTUP_TIME_US)),
+  0xFF,0xFF,
+  0xFFFFFFFF,
+  0xFF,
+  INT_TO_BCD(YEAR),INT_TO_BCD(MONTH),INT_TO_BCD(DAY),
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF
+
+};
+
+#elif BLUENRG_CONFIG == BLUENRG_16_MHZ_RO
+
+const IFR_config_TypeDef IFR_config = {
+#if BLUENRG_MS
+#if SMPS_FREQUENCY == SMPS_4MHz
+  0x02,0x3A,0x40,0x02,
+  0x34,0x1B,0x02,0x39,
+  0xA2,0x02,0x3C,0x20,
+  0x00,0xFF,0xFF,0xFF,
+#elif SMPS_FREQUENCY == SMPS_8MHz
+  0x02,0x3A,0x40,0x02,
+  0x34,0x1B,0x02,0x39,
+  0xAE,0x00,0xFF,0xFF,
+  0x00,0xFF,0xFF,0xFF,
+#else
+#error Incorrect SMPS_FREQUENCY
+#endif /* SMPS_FREQUENCY */
+#else
+  0x02,0x3A,0x58,0x02,
+  0x39,0xA2,0x02,0x34,
+  0x1B,0x00,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+#endif /* BLUENRG_MS */
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+
+  0x02,0x1C,0x43,0x02,
+  0x20,0xEC,0x02,0x1F,
+  0xAF,0x00,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+  0xFF,0xFF,0xFF,0xFF,
+
+  STACK_MODE,
+  0xFF,0xFF,0xFF,
+  0xFFFFFFFF,0xFFFFFFFF,0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  htobs(0x01F4),
+  0x00,
+  0xFF,
+  htobs(FROM_US_TO_SYS_TIME(HS_STARTUP_TIME_US)),
+  0xFF,0xFF,
+  0xFFFFFFFF,
+  0xFF,
+  INT_TO_BCD(YEAR),INT_TO_BCD(MONTH),INT_TO_BCD(DAY),
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF,
+  0xFFFFFFFF
+};
+
+#elif BLUENRG_CONFIG == BLUENRG_CUSTOM_CONFIG
+/* Copy and paste here your custom IFR_config structure. It can be generated
+ * with BlueNRG GUI.
+ */
+#else
+#warning "BLUENRG_CONFIG not valid. IFR can't be used." 

```

Comment:
It should be fine to document IFR usage as the warning is raised, in the Readme for example

Reply:
Clarification: we left this functionality in the library because it is present in the STM32CubeExpansion but we didn't test it. 
Maybe it is better to remove it. What do you think?

Reply:
kept as it is.